src/main/java/hudson/plugins/copyartifact/ParametersBuildFilter.java

```diff
@@ -66,7 +68,8 @@ public boolean isValid(Job<?,?> job) {
                 }
             }
             return true;
-        } catch (Exception ignore) {
+        } catch (InterruptedException ignore) {
+        } catch (IOException ignore) {
```

Comment:
```
    <BugInstance type='REC_CATCH_EXCEPTION'
        priority='Normal'
        category='STYLE'
        message='Exception is caught when Exception is not thrown in hudson.plugins.copyartifact.ParametersBuildFilter.isValid(Job)'
        lineNumber='69'/>
```

Anyway, this method is no longer used (6243f6bcd5ebe872d3c03c8d740b45e3d714f1c4).